### DISCUSSION TITLE
How to get an overview of your activity on your profile?

### DISCUSSION THREAD

**Discussion by SangeethaPrabhu2021** [community/community#69009] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I need to analyze my commits, issues, pull requests, and code reviews. What are the possible steps?

**Comment by 0xTanzim** [comment]
To get started, you need to create a repository with your username and add a Readme.md file. There are several websites where you can configure your Readme.md file. You can take a look at my repository using the following link: https://github.com/TanzimHossain2/TanzimHossain2.git. Once you have edited your profile, it should look similar to mine. For editing, I used this website: https://gprm.itsvg.in/
![image](https://github.com/community/community/assets/60542516/52178e8d-8278-4363-b5f9-b11c044e55c9)

**Comment by mixed-farming** [comment]
Steps: 
1) (Skip this step if you are already at _github.com/username_) Navigate to the upper-right corner of GitHub.com and click on your profile picture, followed by selecting "Your profile."
2) Right above your contributions graph, access the Contribution settings dropdown menu, and proceed to click on "Activity overview."

For more information, refer to [this](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-an-overview-of-your-activity-on-your-profile)

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
